src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/singlespaceseparator/InputSingleSpaceSeparatorErrors.java

```diff
@@ -1,9 +1,9 @@
-package  com.puppycrawl.   tools.checkstyle.checks.whitespace;
+package  com.puppycrawl.   tools.checkstyle.checks.whitespace.singlespaceseparator;
 
 import java.util.List;
 import  java.util.Vector;
 
-public class      InputSingleSpaceErrors  {
+public class      InputSingleSpaceSeparatorErrors  {
```

Comment:
Ummm. After I did this, It gives me a `COMPILATION ERROR` and told me to change the file name to InputSingleSpaceErrors.java